cirq/ops/pauli_string.py

```diff
@@ -856,8 +857,8 @@ def inline_times_pauli_string_like(self,
         if isinstance(contents, PauliString):
             # Note: cirq.X/Y/Z(qubit) are PauliString instances.
             self.inline_times_pauli_string(contents)
-        elif gate_operation.op_gate_isinstance(contents,
-                                               common_gates.IdentityGate):
+        elif (isinstance(contents, raw_types.Operation) and
+              isinstance(contents.gate, common_gates.IdentityGate)):
```

Comment:
The point of `op_gate_isinstance` was to remove the need for a double `isinstance` check.  With the new Operation.gate property, this check should just be `isinstance(op.gate, ...)`.

Reply:
`contents` is a `PAULI_STRING_LIKE`, not an `Operation`, which is why we need the double check.

Reply:
I see.  I liked `op_gate_isinstance`.